### ISSUE TITLE
Disable gutter highlights

### ISSUE THREAD

**Issue by AntonioRedondo** [streetsidesoftware/vscode-spell-checker#504]
Hi there,

Thank you for this great extension.

I'm looking for a feature that seems isn't implemented yet. Would it be possible to add a checkbox on the extension settings to disable typo highlights on the right-hand side gutter?

My gutter already has enough highlights with git changes and code warning and errors. Typos highlights work just fine when are only visible as underlined words.

**Comment by YayC** [comment]
I was also hoping for this setting. Would be great to get rid of those gutter annotations for legacy projects.

**Comment by chodorowicz** [comment]
Yeah, gutter with the spell check highlights becomes to cluttered to be usable. Being able to turn them of would be great.

**Comment by Jason3S** [comment]
I don't think it is possible to set Diagnostic colors per extension. Related discussion: [#128 Highlight color](https://github.com/streetsidesoftware/vscode-spell-checker/issues/128)
